### PR TITLE
Remove log statement from s3.py

### PR DESCRIPTION
**Pull request by terrazoon**
## Description

A log statement with level INFO was left in the s3.py code that retrieves phone numbers from S3.  While there is no info disclosure problem, there is a performance problem as a lot of work is going on this part of the code.  Not only does the logging literally slow down the processing, it also clutters the logs with unhelpful (99.9% of the time) messages.  

## Security Considerations

N/A

### CODE REVIEW

**Review by A-Shumway42 — APPROVED** [review]
Will definitely help in reading through the logs!

File: app/aws/s3.py [hunk 1]
```diff
@@ -82,27 +82,24 @@ def incr_jobs_cache_misses():
         redis_store.set(JOBS_CACHE_MISSES, 1)
     else:
         redis_store.incr(JOBS_CACHE_MISSES)
-    hits = redis_store.get(JOBS_CACHE_HITS).decode("utf-8")
-    misses = redis_store.get(JOBS_CACHE_MISSES).decode("utf-8")
-    current_app.logger.debug(f"JOBS CACHE MISS hits {hits} misses {misses}")
+    redis_store.get(JOBS_CACHE_HITS).decode("utf-8")
+    redis_store.get(JOBS_CACHE_MISSES).decode("utf-8")
```
Comment (ccostino) [inline app/aws/s3.py:86]: Do we still need the `get` statements here if we're not doing anything with the value?  All that needs to happen is the `set`, right?

File: app/aws/s3.py [hunk 2]
```diff
@@ -82,27 +82,24 @@ def incr_jobs_cache_misses():
...
 def incr_jobs_cache_hits():
     if not redis_store.get(JOBS_CACHE_HITS):
         redis_store.set(JOBS_CACHE_HITS, 1)
     else:
         redis_store.incr(JOBS_CACHE_HITS)
-    hits = redis_store.get(JOBS_CACHE_HITS).decode("utf-8")
-    misses = redis_store.get(JOBS_CACHE_MISSES).decode("utf-8")
-    current_app.logger.debug(f"JOBS CACHE HIT hits {hits} misses {misses}")
+    redis_store.get(JOBS_CACHE_HITS).decode("utf-8")
+    redis_store.get(JOBS_CACHE_MISSES).decode("utf-8")
```
Comment (ccostino) [inline app/aws/s3.py:95]: Same question as above:  do we still need the `get` statements here if we're not doing anything with the value?  All that needs to happen is the `set`, right?

**Review by ccostino — COMMENTED** [review]
Thanks, @terrazoon!  Just one quick thing I wanted to check on before approving here.

**Review by ccostino — APPROVED** [review]
Thanks, @terrazoon!

**Pull request merged**